packages/react-centra-checkout/src/ApiClient/index.ts

```diff
@@ -5,7 +5,7 @@ class ApiClient {
 
   options: RequestInit
 
-  static default: ApiClient
+  private static _default?: ApiClient
```

Comment:
Are we sure this should be private? What if you want to modify the default client instead of bringing your own?

Reply:
We'll use the getters to ensure both instance and class access will lead to the same instance of `ApiClient.default`.